[2022-05-06 11:13:38,342] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_3 manual__2022-05-06T05:43:03.540785+00:00 [queued]>
[2022-05-06 11:13:38,346] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_3 manual__2022-05-06T05:43:03.540785+00:00 [queued]>
[2022-05-06 11:13:38,346] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-05-06 11:13:38,346] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-05-06 11:13:38,346] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-05-06 11:13:38,361] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_3> on 2022-05-06 05:43:03.540785+00:00
[2022-05-06 11:13:38,363] {standard_task_runner.py:52} INFO - Started process 1464602 to run task
[2022-05-06 11:13:38,364] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_3', 'manual__2022-05-06T05:43:03.540785+00:00', '--job-id', '9', '--raw', '--subdir', '/home/sanjeev/Documents/SubDAGs/dags/paralle_processing.py', '--cfg-path', '/tmp/tmpueag2128', '--error-file', '/tmp/tmppfn9dx71']
[2022-05-06 11:13:38,365] {standard_task_runner.py:77} INFO - Job 9: Subtask task_3
[2022-05-06 11:13:38,402] {logging_mixin.py:109} INFO - Running <TaskInstance: parallel_dag.task_3 manual__2022-05-06T05:43:03.540785+00:00 [running]> on host kumar
[2022-05-06 11:13:38,438] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2022-05-06T05:43:03.540785+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-06T05:43:03.540785+00:00
[2022-05-06 11:13:38,439] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-06 11:13:38,439] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 5']
[2022-05-06 11:13:38,448] {subprocess.py:85} INFO - Output:
[2022-05-06 11:13:43,451] {subprocess.py:93} INFO - Command exited with return code 0
[2022-05-06 11:13:43,471] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_3, execution_date=20220506T054303, start_date=20220506T054338, end_date=20220506T054343
[2022-05-06 11:13:43,486] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-06 11:13:43,497] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
